### PR TITLE
[bombastic perks] cleaned up prereqs stuff

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Mods "cleaned up prereqs"

#### Purpose of change
Set up some standards for the style  of perks added in #64777 

#### Describe the solution
Added docs, migrated to a single entry with logic that respects mod settings.

#### Testing
Works in game

#### Additional context
@TheShadowFerret if you are interested in seeing what I cooked up for any of your additional perks, if they have reqs they should have the format in the doc in this PR. This way a player can view them (even if they can't select them), and there's a notification if it doesn't work.

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
>@TheShadowFerret if you are interested in seeing what I cooked up for any of your additional perks, if they have reqs they should have the format in the doc in this PR. This way a player can view them (even if they can't select them), and there's a notification if it doesn't work.

I shall rewrite the two I made & retracted for these 🦦

File: data/mods/BombasticPerks/perkmenu.json [hunk 1]
```diff
@@ -51,61 +51,17 @@
...
+            "or": [
+              { "u_has_trait": "perk_way_openpalm" },
+              { "u_has_trait": "perk_way_closedfist" },
+              { "u_has_trait": "perk_way_pinchedfingers" }
+            ]
+          }
         },
-        "text": "Gain [<trait_name:perk_way_openpalm>]",
+        "text": "Gain [Way Of The Fist]",
         "topic": "TALK_PERK_MENU_OPENPALM"
```
Comment (TheShadowFerret) [inline data/mods/BombasticPerks/perkmenu.json:63]: Shouldn't this topic be renamed if it leads to more than just Way of the Open Palm 🤔

Comment (bombasticSlacks) [reply data/mods/BombasticPerks/perkmenu.json:63]: probably, yeah

**Comment by TheShadowFerret** [comment]
Do mutually exclusive perks count as variants if they're just the reverse of each other?

(Asking for an anti-Bloody Mess perk for skin & bionic harvesters)

**Comment by bombasticSlacks** [comment]
> Do mutually exclusive perks count as variants if they're just the reverse of each other?
(Asking for an anti-Bloody Mess perk for skin & bionic harvesters)

I think, if possible, anything that is mutually exclusive should be on the same inner menu.

**Comment by TheShadowFerret** [comment]
> I think, if possible, anything that is mutually exclusive should be on the same inner menu.

Should perks that 'stack'(ex. Stronger leads to Stronger...er 😛 I'm not going to PR that) or require some related perk also be in the same inner menu?

**Comment by bombasticSlacks** [comment]
> Should perks that 'stack'(ex. Stronger leads to Stronger...er 😛 I'm not going to PR that) or require some related perk also be in the same inner menu?

I can't imagine a situation where that is necessary, and would prefer it's never the case, but in that case, for clarity they would have to be separate entries.

**Pull request merged**